usagov/vote-gov-drupal
# Vote-1130: Life Events Refinements

## Jira ticket

[Vote-1130](https://cm-jira.usa.gov/browse/VOTE-1130)

## Description

Style Refinements for life events section on homepage.

Changes applied
- updated content to match design 
- updated link to added hover effect 
- adjusted column use and offset 
- adjusted padding for break points 

[Figma Link](https://www.figma.com/design/u82ICmmHNBKpE1Tw4nsO0K/Vote.gov-Redesign-2024-(Master)?node-id=8012-20081&m=dev)

## Deployment and testing

### Post-deploy steps

1. run `lando retune` then cd into `votegov` theme and run `npm run build`

### QA/Testing instructions

1. log into site and update the quick links section content to match design -> Heading: Have you changed your name, address, or party affiliation?, Content: You'll select your state or territory to get started., Link: Url `/register` Link Text - Check or update your registration
2. [figma Link](https://www.figma.com/design/u82ICmmHNBKpE1Tw4nsO0K/Vote.gov-Redesign-2024-(Master)?node-id=8012-20081&m=dev) for reference - review the default section on the far left at the various break points -> Mobile, tablet and desktop 
3. check that hover effect is working as expected (adding underline and arrow moves from 4px to 12px)
4. check that no major visual regressions appear 

## Checklist for the Developer




## Checklist for the Peer Reviewers
